srcalc/src/test/java/gov/va/med/srcalc/vista/RpcVistaPatientDaoTest.java

```diff
@@ -67,4 +77,53 @@ public final void testSaveNoteSuccess() throws Exception
     	        SaveNoteCode.SUCCESS,
     	        dao.saveRiskCalculationNote(PATIENT_DFN, ELECTRONIC_SIGNATURE, DUMMY_BODY));
     }
+    
+    @Test
+    public final void testLabRetrievalFailure() throws Exception
+    {
+        final List<String> labNames = new ArrayList<String>();
+        labNames.add("ALBUMIN");
+        final VistaProcedureCaller caller = mock(VistaProcedureCaller.class);
+        rpcSetup(caller);
+        // Anything besides a valid measurement is returned as an empty string
+        when(caller.doRetrieveLabs(eq(RADIOLOGIST_DUZ), eq(String.valueOf(PATIENT_DFN)), anyListOf(String.class)))
+            .thenReturn("");
+        final RpcVistaPatientDao dao = new RpcVistaPatientDao(caller, RADIOLOGIST_DUZ);
+        // Behavior verification
+        final Patient patient = dao.getPatient(PATIENT_DFN);
+        assertEquals(new HashMap<String, RetrievedValue>(), patient.getLabs());
+    }
+    
+    @Test
+    public final void testLabRetrievalSuccess() throws Exception
+    {
+        final List<String> labNames = new ArrayList<String>();
+        labNames.add("ALBUMIN");
```

Comment:
Why not just use `LabRetrievalEnum.ALBUMIN.getPossibleLabNames()` here to avoid repetition?